docs/stratis.txt

```diff
@@ -221,6 +221,19 @@ Alerts::
           Any unusual or urgent information about the pool of which the
           user should be made aware.
 
+FIELDS for stratis pool list --stopped
+
+Name::
+	  The name of the pool.
+UUID::
+	  The UUID of the pool.
+# Devices::
+	  The number of devices used by the pool.
+Key Description::
+	  The key description used by the pool.
+Clevis::
+	  The Clevis binding used by the pool.
```

Comment:
This is only whether or not the pool uses Clevis encryption. If yes, the string is "present" if no, "N/A".

Reply:
Resolved.